en/index.md

```diff
@@ -407,6 +407,51 @@ rather than plain English. Try to keep it simple.
 #21 No double space after a period/full stop
 -->
 
+## Admonitions
+
+Admonitions (also commonly referred to as "admonishments" or sometimes
+"callouts" or "Notifications") are a device used in documentation to draw attention to a
+particular statement or paragraph of text. Typically their use is to highlight:
+
+ - A consequence of performing a particular action
+ - An additional source of information which is useful but not required
+ - A helpful tip that will save effort/time
+ - A reminder of some pre-requisite or restriction
+
+ While this style guide usually concerns itself solely with the intent and
+ structure of content rather than the published form of that content, there is
```

Comment:
I'm not sure this is necessary, or perhaps it belongs in a dedicated section that explains the scope of the style guide and what type of material is included (that's not a recommendation!).

I know what you mean here, but I'm not sure if the choice to use a callout is obviously distinct from concerns relating to intent or structure, and I don't think this section needs to explicitly justify its own inclusion.